docs/whats-new.md

```diff
@@ -0,0 +1,196 @@
+  - v2 and v3 swagger API specifications comparison generated by [swagger-diff](https://github.com/Sayi/swagger-diff) tool: [v2v3Changelog.html](media/v2v3Changelog.html)
+### Database
+  - The v2 and v3 databases have structural differences; any existing v2 database should be upgraded before using by Platform v3 
+  - Automatic migration scripts were added to upgrade from v2 database to v3
+
+## Current v3 modules
+
+|  Id      |      Title    | Version |
+|:---------|:-------------:|:-------:|
+| [VirtoCommerce.Core](https://virtocommerce.com/apps/extensions/virto-core-module) | Commerce core module | 3.0.0 |
+| [VirtoCommerce.AzureSearch](https://virtocommerce.com/apps/extensions/virto-azure-search-module) | Azure Search module | 3.0.0 |
+| [VirtoCommerce.Cart](https://virtocommerce.com/apps/extensions/virto-shoppingcart-module) | Shopping cart module | 3.0.0 |
+| [VirtoCommerce.Catalog](https://virtocommerce.com/apps/extensions/virto-catalog-module) | Catalog module | 3.0.0 |
+| [VirtoCommerce.Content](https://virtocommerce.com/apps/extensions/virto-cms-module) | CMS Content module | 3.0.0 |
+| [VirtoCommerce.Customer](https://virtocommerce.com/apps/extensions/virto-customer-module) | Customer management module | 3.0.0 |
+| [VirtoCommerce.ElasticSearch](https://github.com/VirtoCommerce/vc-module-elastic-search) | Elastic Search module | 3.0.0 |
+| [VirtoCommerce.Export]() | Export module | 3.0.0 |
+| [VirtoCommerce.ImageTools](https://virtocommerce.com/apps/extensions/image-tools-module) | Image tools module | 3.0.0 |
+| [VirtoCommerce.Inventory]() | Inventory module | 3.0.0 |
+| [VirtoCommerce.LuceneSearch](https://virtocommerce.com/apps/extensions/virto-lucene-search-module) | Lucene Search module | 3.0.0 |
+| [VirtoCommerce.Marketing](https://virtocommerce.com/apps/extensions/virto-marketing-module) | Marketing module | 3.0.0 |
+| [VirtoCommerce.Notifications](https://virtocommerce.com/apps/extensions/virto-module-notification) | Notifications module | 3.0.0 |
+| [VirtoCommerce.Orders]() | Orders module | 3.0.0 |
+| [VirtoCommerce.Payment](https://virtocommerce.com/apps/extensions/virto-payment-module) | Payment module | 3.0.0 |
+| [VirtoCommerce.Pricing]() | Pricing module | 3.0.0 |
+| [VirtoCommerce.Search](https://virtocommerce.com/apps/extensions/virto-core-module) | Search module | 3.0.0 |
+| [VirtoCommerce.Shipping](https://virtocommerce.com/apps/extensions/virto-shipping-module) | Shipping module | 3.0.0 |
+| [VirtoCommerce.Sitemaps](https://virtocommerce.com/apps/extensions/automatic-sitemaps-module) | Sitemaps module | 3.0.0 |
+| [VirtoCommerce.Store](https://virtocommerce.com/apps/extensions/virto-store-module) | Store module | 3.0.0 |
+| [VirtoCommerce.Subscription]() | Subscription module | 3.0.0 |
+| [VirtoCommerce.Tax](https://virtocommerce.com/apps/extensions/virto-tax-module) | Tax module | 3.0.0 |
```

Comment:
I found the broken links. Ex: Current v3 modules
https://virtocommerce.com/apps/extensions/virto-tax-module

@eMazeika  @tatarincev  Maybe, better to replace with direct link on GitHub?